README.md

```diff
@@ -36,7 +27,7 @@ Battlesnake documentation can be found at <https://docs.battlesnake.io>.
 
 <a id="org3ada019"></a>
 
-## Strategy
+## About
```

Comment:
@austinben Can we add some basic info about the template here.  Maybe about how it's different from the one provided by battlesnake.io.  Particularly the use of `Gradle`, and `Log4j`